### ISSUE TITLE
Packages list for Wiko OEM

### ISSUE THREAD

**Issue by ghost** [0x192/universal-android-debloater#856]
**Wiko Power U20 (W-V750BN-TVM)**

**Packages:**

```
com.ironsoure.appcloud.oobe.wiko
com.ape.wikolegal
com.ape.walkfit
com.wiko.packageinstaller
com.ape.smartgesture
com.ape.cleanassist
com.ape.cleanassistoverlay
com.ape.oneclean
com.ape.fmradio
com.ape.fmradiooverlay
com.wiko.services
com.ape.massageexpress
com.ape.weatherlive
com.ape.weatherliveoverlay
com.ape.gamemode
com.ape.wikosetupwizard
com.ape.soundrecorder
com.ape.soundrecorderoverlay
com.ape.easymode1
```

- [x] **I removed all those packages on my phone**
      If not why. Leave the brackets blank and explain why.

## Document each package the best you can

**List**: `Google`|`Misc`|`OEM` (manufacturer)|`AOSP`|`Pending`|`Carrier` (isp).

**Removal**: `Recommended`, `Advanced`, `Expert` (this can break important features),
or `Unsafe` (this can bootloop the phone or break extremely important features).

### com.ironsoure.appcloud.oobe.wiko

**List**: OEM
**Removal**:Recommended
` Provide first time setup for Wiko mobile `

### com.ape.wikolegal
**List**: OEM
**Removal**: Recommended
` Provide legal for first time setup in Wiko mobile `

### com.ape.walkfit
**List**: OEM
**Removal**: Recommended
` Walk counter and health for Wiko mobile `

### com.wiko.packageinstaller
**List**: OEM 
**Removal**: Recommended
` Provide first time app install for Wiko mobile `

### com.ape.smartgesture
**List**: OEM
**Removal**: Advanced
` Provide smart gesture feature for Wiko mobile `

### com.ape.cleanassist
**List**: OEM
**Removal**: Advanced
` Provide junk cleaner feature for Wiko mobile `

### com.ape.cleanassistoverlay
**List**: OEM
**Removal** Advanced
` Overlay app for com.ape.cleanassist If uninstalled com.ape.cleanassist should be removed together. `

### com.ape.oneclean
**List**: OEM
**Removal**: Advanced
` Provide memory cleaner feature for Wiko mobile `

### com.ape.fmradio
**List**: OEM
**Removal**: Advanced
` FM Radio app for Wiko mobile `

### com.ape.fmradiooverlay
**List**: OEM
**Removal**: Advanced
` Overlay app for com.ape.fmradio If uninstalled com.ape.fmradio should be removed together. `

### com.wiko.services
**List**: OEM
**Removal**: Advanced
` Support app for Wiko mobile `

### com.ape.massageexpress
**List**: OEM
**Removal**: Advanced
` Provide Wiko screen message feature `

### com.ape.weatherlive
**List**: OEM
**Removal**: Advanced
` Weather app for Wiko mobile `

### com.ape.weatherliveoverlay
**List**: OEM
**Removal**: Advanced
` Overlay app for com.ape.weatherlive If uninstalled com.ape.weatherlive should be removed together. `

### com.ape.gamemode
**List**: OEM
**Removal**: Advanced
` Provide game mode feature for Wiko mobile`

### com.ape.wikosetupwizard
**List**: OEM 
**Removal**: Recommended
` Provide first time app install for Wiko mobile `

### com.ape.soundrecorder
**List**: OEM
**Removal**: Advanced
` Sound recorder app for Wiko mobile `

### com.ape.soundrecorderoverlay
**List**: OEM
**Removal**: Advanced
` Overlay app for com.ape.soundrecorder If uninstalled com.ape.soundrecorder should be removed together. `

### com.ape.easymode1
**List**: OEM
**Removal**: Advanced
` Provide easy mode for Wiko mobile `

**Comment by AnonymousWP** [comment]
Because the project seems dead, I'm not going to be contributing to large issues anymore, because it feels like wasting my time. If @0x192 becomes active again, I'll open bigger PRs again, but for now it seems a bit useless.

**Comment by Frigyes06** [comment]
@AnonymousWP I'm trying to revive this project on a fork! 
https://github.com/Frigyes06/universal-android-debloater
Feel free to contribute

**Comment by AnonymousWP** [comment]
> @AnonymousWP I'm trying to revive this project on a fork! https://github.com/Frigyes06/universal-android-debloater Feel free to contribute

Will you also actually build releases and do you have decent Rust knowledge?

**Comment by Frigyes06** [comment]
> > @AnonymousWP I'm trying to revive this project on a fork! https://github.com/Frigyes06/universal-android-debloater Feel free to contribute
> 
> Will you also actually build releases and do you have decent Rust knowledge?

Yeah! I can build it, and I'm actually looking to improve my rust skills, so this project is awesome for me.

**Comment by AnonymousWP** [comment]
> > > @AnonymousWP I'm trying to revive this project on a fork! https://github.com/Frigyes06/universal-android-debloater Feel free to contribute
> > 
> > 
> > Will you also actually build releases and do you have decent Rust knowledge?
> 
> Yeah! I can build it, and I'm actually looking to improve my rust skills, so this project is awesome for me.

Great to hear. Could you start pushing the PRs of this repo to yours? That'd be a great start and an incentive for me to start contributing again.

**Comment by Frigyes06** [comment]
> > > > @AnonymousWP I'm trying to revive this project on a fork! https://github.com/Frigyes06/universal-android-debloater Feel free to contribute
> > > 
> > > 
> > > Will you also actually build releases and do you have decent Rust knowledge?
> > 
> > 
> > Yeah! I can build it, and I'm actually looking to improve my rust skills, so this project is awesome for me.
> 
> Great to hear. Could you start pushing the PRs of this repo to yours? That'd be a great start and an incentive for me to start contributing again.

I'm trying to encourage people with open PRs here to open them to my repo. I feel it's unfair for me to copy their contributions over into my own PRs. Of course, if they won't reply soon, I'll solve the mentioned issues myself and open PRs. I'm currently setting up code scanning and going through deps to make sure it doesn't have any blaring security flaws.

**Comment by AnonymousWP** [comment]
> > > > > @AnonymousWP I'm trying to revive this project on a fork! https://github.com/Frigyes06/universal-android-debloater Feel free to contribute
> > > > 
> > > > 
> > > > Will you also actually build releases and do you have decent Rust knowledge?
> > > 
> > > 
> > > Yeah! I can build it, and I'm actually looking to improve my rust skills, so this project is awesome for me.
> > 
> > 
> > Great to hear. Could you start pushing the PRs of this repo to yours? That'd be a great start and an incentive for me to start contributing again.
> 
> I'm trying to encourage people with open PRs here to open them to my repo. I feel it's unfair for me to copy their contributions over into my own PRs. Of course, if they won't reply soon, I'll solve the mentioned issues myself and open PRs. I'm currently setting up code scanning and going through deps to make sure it doesn't have any blaring security flaws.

Hereby you have my permission to push my PRs on this repo to your repo, and I think either of these two should work:

- https://stackoverflow.com/questions/38918942/pull-request-from-a-different-repository
- https://github.com/orgs/community/discussions/49745

**Comment by Frigyes06** [comment]
> > > > > > @AnonymousWP I'm trying to revive this project on a fork! https://github.com/Frigyes06/universal-android-debloater Feel free to contribute
> > > > > 
> > > > > 
> > > > > Will you also actually build releases and do you have decent Rust knowledge?
> > > > 
> > > > 
> > > > Yeah! I can build it, and I'm actually looking to improve my rust skills, so this project is awesome for me.
> > > 
> > > 
> > > Great to hear. Could you start pushing the PRs of this repo to yours? That'd be a great start and an incentive for me to start contributing again.
> > 
> > 
> > I'm trying to encourage people with open PRs here to open them to my repo. I feel it's unfair for me to copy their contributions over into my own PRs. Of course, if they won't reply soon, I'll solve the mentioned issues myself and open PRs. I'm currently setting up code scanning and going through deps to make sure it doesn't have any blaring security flaws.
> 
> Hereby you have my permission to push my PRs on this repo to your repo, and I think either of these two should work:
> 
>     * https://stackoverflow.com/questions/38918942/pull-request-from-a-different-repository
> 
>     * https://github.com/orgs/community/discussions/49745

I figured out a way to do it while keeping commit history intact. Started making the PRs to the fork
